loops_lab.js

```diff
@@ -0,0 +1,187 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+let num = 10;
+while(num > 2){
+  num--;
+  console.log(num);
+
+}
+
+for(num = 10 - 1;num > 1;num--){
+  console.log(num);
+}
+
+/* 2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
+For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even") */
+num = 10
+let i = 0;
+while(i <= num){
+
+  if(i%2 === 0){
+    console.log(i + "is even")
+  }
+  else{
+    console.log(i + "is odd")
+  }
+i++
+}
+
+for(i = 0; i <= num;i++){
+  if(i%2 === 0){
+    console.log(i + "is even")
+  }
+  else{
+    console.log(i + "is odd")
+  }
+}
+
+/* 3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
+For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
+
+_Bonus_ think of another way to solve it.
+  <details>
+    <summary>
+      Hint
+    </summary>
+    Find the final number and increment the loop by 9.
+  </details> */
+
+  num = 10
+  i = 0;
+  while(i <= num){
+
+  console.log( i + " * 9 = " + i*9)
+  i++
+  }
+
+  for(i = 0; i <= num;i++){
+    console.log( i + " * 9 = " + i*9);
+  }
+
+//Write code that prints/logs all the numbers that end in a 5 from 1 to 100, exclusive.
+for(i = 5; i <= 100;i+=10){
+console.log(i)
+
+}
+
+/*Without running/executing your code, how many times will the loop below run? Explain why.
+
+let i = 5;
+
+while (i > 3) {
+    i += 1
+} */
+
+// Your explanation here
+// infinitly because i never becomes less than 3 the condition that will make the while loop stop
+
+//  Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+for(i = 0; i <= 100;i++){
```

Comment:
When I ran this code, it did not give me FizzBuzz on numbers such as 30 or 75. If statements can definitely be improved in this case. One suggestion is that we can move
 "else if(i%5 === 0 && i%3 === 0 ){
    console.log("FizzBuzz")
  }" 
to the top so that it can catch all cases that are FizzBuzz first then proceed with rest of if statements.